app/views/admin/users/_reports.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="crayons-card p-6">
+  <div>
+    <h2 class="d-inline">Recent Reports</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="pt-6 collapse" id="reactions-row">
+    <% unless @related_reports.empty? %>
+      <% @related_reports.each do |report| %>
+        <a href="<%= admin_feedback_messages_path %>" class="list-group-item list-group-item-action action px-5 d-flex justify-content-between">
+        <span><%= report.category.capitalize %> (<%= report.status %>)</span>
+        <span><strong><%= report.feedback_type.titlecase %></strong>: <%= report.message %></span>
```

Comment:
So we technically only have one `feedback_type` now, and it's become an unused column.

Instead, I think it's more useful to show if it's a reporter, offender, or affected report. I think the easiest way to show this is to do `report.reporter_id == user.id`. You might want to make a method for the instance in the model, like:

```ruby
# feedback_message.rb
def user_type(user_id)
  # returns the string "Reporter, Offender, and Affected" if all three IDs are true
end
```